.github/workflows/dart-desktop-tests.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - name: Publish realm_dart coverage
         if: inputs.os == 'linux'
         id: publish-coverage
-        uses: coverallsapp/github-action@a1a1a8a300a7e89df3630639df8fb23de5cc6368
+        uses: coverallsapp/github-action@v2
```

Comment:
Why not use a specific hash? We usually prefer specific hashes to avoid a possible supply chain attack, though I agree we rarely do any a thorough validation when upgrading.